payments/admin.py

```diff
@@ -1,6 +1,29 @@
+import json
+
 from django.contrib import admin
-from payments.models import StripeAccount, StripeEvent
+from .models import StripeAccount, StripeEvent
+
+
+class StripeAccountAdmin(admin.ModelAdmin):
+
+    readonly_fields = (
+        'user', 'account_id', 'secret_key', 'public_key',
+        'available_balance'
+    )
+
+
+class StripeEventAdmin(admin.ModelAdmin):
+    list_display = (
+        'event_id', 'type', 'created', 'verified', 'processed', 'user_id')
+    search_fields = ['message_text']
+
+    readonly_fields = (
+        'type', 'created', 'event_id', 'processed', 'message_text',)
 
+    def message_text(self, instance):
```

Comment:
And if we're worried about displaying secrets in the django admin interface, maybe we want to strip any secrets out of message before we display it?